components/hab/src/main.rs

```diff
@@ -675,7 +679,6 @@ fn exec_subcommand_if_called(ui: &mut UI) -> Result<()> {
         ("sup", _, _) => command::sup::start(ui, env::args_os().skip(2).collect()),
         ("start", _, _) => command::launcher::start(ui, env::args_os().skip(1).collect()),
         ("stop", _, _) => command::sup::start(ui, env::args_os().skip(1).collect()),
-        ("svc", "start", _) => command::launcher::start(ui, env::args_os().skip(2).collect()),
```

Comment:
The plan would be to move all of these into the normal flow of the hab `start` function.